src/ReleaseHistory.md

```diff
@@ -14,6 +14,7 @@
 - UEE => eliminate unhandled exceptions in engine
 
 ## **v4.1.0** UNRELEASED
+* BUG: Fix PDB probing details incompleteness on passing `PdbLoad` value to `--trace` command-line argument. [828](https://github.com/microsoft/binskim/pull/828)
```

Comment:
>BUG: Fix PDB probing details incompleteness on passing `Pdb [](http://example.com/codeflow?start=2&length=59)

* BUG: Probe local symbols directory for PDBs in all code paths. [828](https://github.com/microsoft/binskim/pull/828)
* BUG: Add missing output in PDB load tracing (enabled by `--trace PdbLoad`. [828](https://github.com/microsoft/binskim/pull/828)